taskcluster/ci/signing/kind.yml

```diff
@@ -7,6 +7,7 @@ loader: mozillavpn_taskgraph.loader.multi_dep:loader
 transforms:
     - mozillavpn_taskgraph.transforms.multi_dep:transforms
     - mozillavpn_taskgraph.transforms.signing:transforms
+    - mozillavpn_taskgraph.transforms.release_index:transforms
```

Comment:
I'm kind of on the fence about this comment, so feel free to leave this as is..

But the `task` transforms support a (undocumented) feature called `index_builders`, e.g:
https://github.com/taskcluster/taskgraph/blob/main/src/taskgraph/transforms/task.py#L890

So basically you could create a "release_index" builder. You'd want to define it somewhere that gets imported from [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/taskcluster/mozillavpn_taskgraph/__init__.py#L21) (you'd likely want to add a new module called "index_builders" or something like that).

Then rather than have your tasks import the transform, you'd add something like:
```
index:
    type: release
    product: mozillavpn
    job-name: build
```
to its definition.

That said, the transform approach works too and is probably easier to trace, so I don't mind going with that approach either.

Reply:
That is super cool buuut i'm afraid, qa would like the index route ready tommorow (eu) morning, so i would probably look into this in a followup, filed a bug to look into that later :)